modules/persistent-storage-csi-migration-enable.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+[id="persistent-storage-csi-migration-enable_{context}"]
+= Enabling CSI automatic migration
+
+[NOTE]
+====
+Enabling CSI automatic migration drains, and then restarts, all nodes in the cluster in sequence. This might take some time.
+====
+
+.Procedure
+
+* Enable feature gates (see xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc[Enabling features using feature gates]).
++
+[IMPORTANT]
+====
+After turning Technology Preview features on using feature gates, they cannot be turned off and cluster upgrades are prevented.
+====
++
+The following configuration example enables CSI automatic migration to both CSI drivers (AWS EBS and Cinder):
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  name: cluster
+spec:
+  featureSet: TechPreviewNoUpgrade <1>
+...
+----
+<1> Enables automatic migration for AWS EBS and Cinder.
++
+You can specify CSI automatic migration for a selected CSI driver by setting `CustomNoUpgrade` `featureSet` and for `featuregates` either:
```

Comment:
This is a really long single step. I'd consider breaking it up into two separate options:

```
* Enable CSI automatic migration by enabling a feature set
* Enable CSI automatic migration for a selected CSI driver
```

You could find better wording I'm sure, but the point being that you can either use a default feature set that includes multiple things, or you can do a custom one to just get the driver you want

Reply:
The problem there is that the first step is enabling feature gates, and then the choice of how you do it, so you end up repeating that 1st step/prerequisite. Bob and I went round and round on this for a long time, and this is kinda the best we could come up with. Bob also consulted other writers on Slack for opinions on this, so I understand your point, but it's one of those difficult to make perfect situations.